### DISCUSSION TITLE
编译报错

### DISCUSSION THREAD

**Discussion by 596142041** [Boos4721/openwrt#29] [Q&A/答疑解惑] [answered]
/home/lpr/openwrt/staging_dir/toolchain-x86_64_gcc-11.2.0_musl/lib/gcc/x86_64-openwrt-linux-musl/11.2.0/../../../../x86_64-openwrt-linux-musl/bin/ld: /home/lpr/openwrt/build_dir/target-x86_64_musl/node-v14.18.0/out/Release/obj.target/tools/v8_gypfiles/libv8_base_without_compiler.a: error adding symbols: malformed archive
这是编译最终的日志,能帮忙看一下是什么原因导致的么

**Comment by Boos4721** [comment, marked as answer]
**Node.js compilation error:**
> error adding symbols: malformed archive
> collect2: error: ld returned 1 exit status
Enter `ulimit -N 2048` on the command line and recompile.
